float32 v_battery
float32 v_pi
float32 v_aux
float32 i_pi
float32 i_aux
float32 pcb_temp
float32 cpu_temp
